### PR TITLE
Update subscriptions.md

### PR DESCRIPTION
**Pull request by batraprashant**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

File: docs/platform/get-started/subscriptions.md [hunk 1]
```diff
@@ -21,14 +21,14 @@ Every user who contributes to software development, deployment, operations, opti
...
-- Continuous Integration (CI)
-  - All users on the Harness Platform, who have permissions to access Harness CI module, will be considered as CI Developers and will consume a license.
-- Internal Developer Portal (IDP)
-  - All users on the Harness Platform, who have permissions to access Harness IDP module, will be considered as IDP Developers and will consume a license.
-- Software Engineering Insights (SEI)
-  - All users read from a configured SCM or Issue Management tool, will be considered SEI Developers and will consume a License.
+#### Code Repository (CR)
+All users on the Harness Platform, who have permissions to access Harness CR module, will be considered as CR Developers and will consume a license.
+#### Continuous Integration (CI)
+All users on the Harness Platform, who have permissions to access Harness CI module, will be considered as CI Developers and will consume a license.
```
Comment (brian-f-harness) [inline docs/platform/get-started/subscriptions.md:27]:
```suggestion
All users who have permission to access the Harness CI module on the Harness Platform will be designated as CI Developers and will consume a license.
```

File: docs/platform/get-started/subscriptions.md [hunk 2]
```diff
@@ -21,14 +21,14 @@ Every user who contributes to software development, deployment, operations, opti
...
+#### Internal Developer Portal (IDP)
+All users on the Harness Platform, who have permissions to access Harness IDP module, will be considered as IDP Developers and will consume a license.
```
Comment (brian-f-harness) [inline docs/platform/get-started/subscriptions.md:29]:
```suggestion
All users who have permission to access the Harness IDP module on the Harness Platform will be designated as IDP Developers and will consume a license.
```

File: docs/platform/get-started/subscriptions.md [hunk 3]
```diff
@@ -21,14 +21,14 @@ Every user who contributes to software development, deployment, operations, opti
...
+#### Software Engineering Insights (SEI)
+All users read from a configured SCM or Issue Management tool, will be considered SEI Developers and will consume a License.
```
Comment (brian-f-harness) [inline docs/platform/get-started/subscriptions.md:31]:
```suggestion
All users read from a configured SCM or Issue Management tool will be considered SEI Developers and will consume a License.
```

**Comment by brian-f-harness** [comment]
@batraprashant Are you okay with the suggested changes? If so, I'll commit them and merge this PR. Thanks.

**Comment by batraprashant** [comment]
> @batraprashant Are you okay with the suggested changes? If so, I'll commit them and merge this PR. Thanks.

Yes @brian-f-harness , good with the changes. Please go ahead and merge. Thanks!

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**